src/pytom_tm/matching.py

```diff
@@ -123,6 +127,14 @@ def __init__(
         self.angle_list = angle_list
         self.angle_ids = angle_ids
         self.stats = {'search_space': 0, 'variance': 0., 'std': 0.}
+        if stats_roi is None:
+            self.stats_roi = (
+                slice(0, volume.shape[0]),
+                slice(0, volume.shape[1]),
+                slice(0, volume.shape[2])
```

Comment:
if the intention is to slice nothing, then `slice(None)` is probably better
```suggestion
                slice(None),
                slice(None),
                slice(None)
```

Reply:
good one, that's easier to read as well